### ISSUE TITLE
Default notification always added to UI when using global config with promoted builds

### ISSUE THREAD

**Issue by aripringle** [jenkinsci/hipchat-plugin#69]
I recently upgraded to the hipchat plugin 1.0.0, and started using the "global configuration" option instead of job-specific settings. We like to have notifications on the job builds, as well as our promotion processes (via the Promoted Builds plugin).

When adding the hipchat notifications as a build action to a promotion process, and leaving the defaults intact (no notifications defines), everything works as expected (global notifications are inherited, and notify correctly on promotions).

However, upon reloading the configuration page for the job, a new notification is added for the "Started" process:

![screen shot 2016-03-29 at 4 07 35 pm](https://cloud.githubusercontent.com/assets/1278449/14125555/0c167e0e-f5c9-11e5-9158-415f83bd1e31.png)

This requires the user to remember to delete this job-specific notification every time they edit the configuration, or else it will only have the single notification set, and no longer use the global configurations.

This behavior doesn't happen on the main job configuration; it only seems to occur in the promotion configuration.

**Comment by aldaris** [comment]
Hmm, that looks strange.. I'll have a look at this

**Comment by aripringle** [comment]
Thanks! Let me know if you need any more info on our environment or the steps to reproduce this.

**Comment by aldaris** [comment]
Well, I was able to reproduce it, but I have no clue on what's going wrong. From what I can tell, the promotion configuration gets saved correctly, so chances are that there is a bug in the Jelly side of things, and that's not something that I can debug.
I've also noticed that there is a JS error displayed on Chrome's dev console, but couldn't really figure out what was going on with that either (so not sure if it is even related to this problem).
